code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -410,19 +410,412 @@ datum/reagent/consumable/eggyolk
+		M << "<span class = 'warning'>[spooky_eat]</span>"
+
+/datum/reagent/ectoplasm/on_mob_life(var/mob/living/M as mob)
+	var/spooky_message = pick("You notice something moving out of the corner of your eye, but nothing is there...", "Your eyes twitch, you feel like something you can't see is here...", "You've got the heebie-jeebies.", "You feel uneasy.", "You shudder as if cold...", "You feel something gliding across your back...")
+	if(prob(5))
+		M << "<span class='warning'>[spooky_message]</span>"
+	..()
+	return
+
+/datum/reagent/soybeanoil
+	name = "Space-soybean oil "
+	id = "soybeanoil"
+	description = "An oil derived from extra-terrestrial soybeans."
+	reagent_state = LIQUID
+	color = "#B1B0B0"
+
+/datum/reagent/soybeanoil/on_mob_life(var/mob/living/M as mob)
+	if(prob(5))
+		M.reagents.add_reagent("porktonium",5)
+	..()
+	return
+
+/datum/reagent/hydrogenated_soybeanoil
+	name = "Partially hydrogenated space-soybean oil"
+	id = "hydrogenated_soybeanoil"
+	description = "An oil derived from extra-terrestrial soybeans, with additional hydrogen atoms added to convert it into a saturated form."
+	reagent_state = LIQUID
+	color = "#B1B0B0"
+
+/datum/reagent/hydrogenated_soybeanoil/on_mob_life(var/mob/living/M as mob)
+	if(prob(8))
+		M.reagents.add_reagent("porktonium",5)
 	..()
+	return
+
+/datum/chemical_reaction/hydrogenated_soybeanoil
+	name = "Partially hydrogenated space-soybean oil"
+	id = "hydrogenated_soybeanoil"
+	result = "hydrogenated_soybeanoil"
+	required_reagents = list("soybeanoil" = 1, "hydrogen" = 1)
+	result_amount = 2
+	required_temp = 520
+	mix_message = "The mixture emits a burnt, oily smell."
+
+/datum/reagent/meatslurry
+	name = "Meat Slurry"
+	id = "meatslurry"
+	description = "A paste comprised of highly-processed organic material. Uncomfortably similar to deviled ham spread."
+	reagent_state = LIQUID
+	color = "#EBD7D7"
+
+/datum/chemical_reaction/meatslurry
+	name = "Meat Slurry"
+	id = "meatslurry"
+	result = "meatslurry"
+	required_reagents = list("corn_starch" = 1, "blood" = 1)
+	result_amount = 2
+	mix_message = "The mixture congeals into a bloody mass."
+	mix_sound = 'sound/effects/blobattack.ogg'
+
+/datum/reagent/mashedpotatoes
+	name = "Mashed potatoes"
+	id = "mashedpotatoes"
+	description = "A starchy food paste made from boiled potatoes."
+	reagent_state = SOLID
+	color = "#D6D9C1"
+
+/datum/reagent/gravy
+	name = "Gravy"
+	id = "gravy"
+	description = "A savory sauce made from a simple meat-dripping roux and milk."
+	reagent_state = LIQUID
+	color = "#B4641B"
+
+/datum/chemical_reaction/gravy
+	name = "Gravy"
+	id = "gravy"
+	result = "gravy"
+	required_reagents = list("porktonium" = 1, "corn_starch" = 1, "milk" = 1)
+	result_amount = 3
+	required_temp = 374
+	mix_message = "The substance thickens and takes on a meaty odor."
```

Comment:
Remember kids - when your mum makes gravy, she's actually mixing a fission product with cornstarch and milk.